eslint.config.mjs

```diff
@@ -262,16 +262,15 @@ export default [
       // ESLint recommended rules that we disable.
       'no-inner-declarations': 'off',
 
-      // JSDoc recommended rules that we disable.
+      // JSDoc rules that we enable.
```

Comment:
Well clearly we still disable a few, that comment is a bit confusing

Reply:
I pushed a new commit to fix this.